clients/service-center-client/src/main/java/org/apache/servicecomb/service/center/client/ServiceCenterRegistration.java

```diff
@@ -131,7 +131,14 @@ public void execute() {
           startTask(new RegisterSchemaTask(0));
         } else {
           Microservice newMicroservice = serviceCenterClient.getMicroserviceByServiceId(serviceResponse.getServiceId());
-          dealIsSwaggerDifferent(newMicroservice);
+          boolean swaggerDifferent = isSwaggerDifferent(newMicroservice);
```

Comment:
`if (isSwaggerDifferent(newMicroservice))` use this to avoid many temporary variables.